### ISSUE TITLE
Fails to start on Hyprland

### ISSUE THREAD

**Issue by thomascft** [neovide/neovide#1422]
**Describe the bug**
I've recently mad the switch to wayland, and tried multiple different compositors. So far these are the one's I've tested that work just fine:

- Sway
- River
- Kwin(Wayland Backend)
- Wayfire

**To Reproduce**
1. Start hyperland with command `Hyprland`
2. Run `neovide --nofork`
3. Output ```[wayland-client error] Attempted to dispatch unknown opcode 0 for wl_shm, aborting.
zsh: IOT instruction (core dumped)  neovide --nofork```

**Expected behavior**
Neovide should open.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS 11.4]
 - Neovide Version [e.g. 0.9.0]
 - Neovim Version [e.g. 0.7.2]

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
```TRACE [neovide] Neovide version: 0.9.0
DEBUG [neovide::bridge::command] Starting neovim with: Command { std: "/usr/bin/nvim" "--embed", kill_on_drop: false }``
**Additional context**
Add any other context about the problem here.

**Comment by shaunsingh** [comment]
Closing in favor of https://github.com/neovide/neovide/issues/1394

Tldr we use a fork of winit that has some keyboard/text-input improvements, but that also means we miss out on quite a few fixes merged upstream. Once we start tracking winit upstream then it should fix itself
